### ISSUE TITLE
Multiple profile support?

### ISSUE THREAD

**Issue by whoiswes** [StephenBlackWasAlreadyTaken/NightWatch#75]
As the parent of a T1, as well as a newly diagnosed T1 myself, it would be VERY beneficial to our entire family if multiple Nightscout sites could be entered and tracked into Nightwatch (aka, multiple profiles).

At a minimum, I would like to see:

Profile specific widgets
Profile specific watchfaces
Combination watchfaces (ie, 1, 2, or 3 T1s per watchface)

I know this is a big order, so I'm opening this for discussion purposes and to gauge interest.

I'm trying to make time to learn how to code for Android (probably Phonegap, zero Java experience but a fair amount of Python/PHP/Powershell) and will contribute how/where I can.

Thanks to everyone who has contributed to Nightscout/xDrip/Nightwatch - I don't know where we'd be without all of them!

Wes

**Comment by AdrianLxM** [comment]
The dirty trick so far was to compile two different versions of NightWatch with different names as if they were two different apps. This didn't have a watchface to follow two people, but you can have two different widget. This is how this looks:
![screenshot_2015-12-27-21-43-41](https://cloud.githubusercontent.com/assets/9692866/12703901/b514335a-c84e-11e5-9802-a9466e3c8c2f.png)

Would this help you?

**Comment by whoiswes** [comment]
Yeah, I've seen that screenshot before Adrian, it would work for the phone display, but I've gotta be honest - the glancibility of a smartwatch has completely changed how I monitor my son (I'm not yet monitoring myself yet but it's coming).  I rarely have Nightscout or Nightwatch open on my phone or workstation anymore, I almost exclusively use the watch (especially since raw values were added).  My wife is the same way, albeit with a Pebble, for now.

I'm also thinking for the less-technically-inclined, having an easier way to monitor/manage multiple T1s within one app would be much more user friendly.

Just thinking longer-term as more and more people are exposed to these apps.

Thanks!

Wes

**Comment by jbriley** [comment]
Second this. Have multiple T1s and would be great to have a swipe-able watch face to display both charts and BG.

**Comment by kimberscott** [comment]
Any new progress in this direction? I'm also looking to monitor both my son's data and my own on a watchface (either together or as different profiles I could switch between), but don't have android development experience yet.

**Comment by Richard42GH** [comment]
> The dirty trick so far was to compile two different versions of NightWatch with different names as if they were two different apps. This didn't have a watchface to follow two people, but you can have two different widget. This is how this looks:
> ![screenshot_2015-12-27-21-43-41](https://cloud.githubusercontent.com/assets/9692866/12703901/b514335a-c84e-11e5-9802-a9466e3c8c2f.png)
> 
> Would this help you?

Hi, I know this is a couple of years ago, but this solution allowing 2 widgets is exactly what I'm looking for but I have not been able to achieve it using app cloners. They only clone the nightwatch app, not the widget. Can you help me with deploying the app with a different name in Android, which I assume creates the related widget with a different name also.

Thanks

Richard
